src/content/1.7/modules/sample_modules/order_view_page_new_hooks/view-order-hooks-signature-widget.md

```diff
@@ -0,0 +1,448 @@
+---
+title: Signature card 
+weight: 10
```

Comment:
I believe you need a weight higher than 10 if you want to ensure the correct order with the first page (or lower I'm not sure but different)
If not it's sorted alphabetically, which might work out of luck but better be safe in case we change the title (and even for coming new pages)

Reply:
changed them from 1 to higher in previous commits

Reply:
Yes I saw it, I wasn't reviewing up to date modifs..